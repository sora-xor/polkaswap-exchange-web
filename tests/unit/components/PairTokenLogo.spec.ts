// import { shallowMount, createLocalVue } from '@vue/test-utils'

// import PairTokenLogo from '@/components/PairTokenLogo.vue'
// import { tokens } from '../../mocks/tokens'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// SoramitsuElementsImport(localVue)

// describe('TokenLogo.vue', () => {
//   beforeEach(() => {
//     TranslationMock(PairTokenLogo)
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(PairTokenLogo, {
//       localVue,
//       propsData: {
//         firstToken: tokens[0],
//         secondToken: tokens[1],
//         size: 'mini'
//       }
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('PairTokenLogo test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
