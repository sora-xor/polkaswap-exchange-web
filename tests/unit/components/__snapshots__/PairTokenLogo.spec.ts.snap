// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TokenLogo.vue should renders correctly 1`] = `
<div
  class="pair-logo pair-logo--mini"
>
  <!---->
   
  <!---->
</div>
`;
